content/blog/2024-01-07-INTA/index.md

```diff
@@ -0,0 +1,51 @@
+
+The decree makes it clear that "In the regional operating units, the **balanced participation** of the different sectors of agricultural production, provincial governments, universities and the scientific-technical community **in their management will ensure the correspondence between the activities of the units and local problems.**" 
+
+## Implementation
+
+The INTA has a directive board composed of members of both the public and the private sector and with representation for different actors in the agricultural sector; they define the politics at national level. The national direction executes the planning of the board, assisted by the organization and administration technical areas.
+
+Each Regional and Research Center has a Regional board of director (I was the firs female member of the Natural Resource Research Center representing researcher that work at the experimental stations) that define the strategy ar regional level aligned with the national policy. Finally, the experimental stations and extension agencies have local advisory councils with the representation of local actors of the sector. All these governance body meet every month.  The National and Regional boards participate in the definition of the budget, the long- and medium-term strategic plan and the hiring of personnel. 
+
+For example, a regional board of director consists of:
+
+1 member representing the national board of directors.
+1 member representing the province goverment.
+1 member representing the universities.
+1 member representing the science-academic sector.
+3 member representing the farmers.
+1 member representing the local advisory councils.
+
+## Science by and for the people
+
+_Citizen Science_ definition say that it is research conducted with participation from the general public, or amateur/nonprofessional researchers or participants for science, social science and many other disciplines.[^1]
+
+Most of INTA's work involves citizen participation, for example, many field trials and technology tests are carried out on farmers' farms developed with with them, or with agricultural machinery manufacturers, many projects involve people recording field data and conducting surveys, but in my opinion **letting citizens be part of the leadership of science organizations** is the clearest and strongest signal that you really want them to be part of it and that their voice and opinion will be taken into account. 
+
+It is not easy, there are frictions and conflicting interests among the sectors represented, but what better than to have them at the same table, dialoguing and agreeing on what is best for all.  What is better than knowing that your research and extension efforts are serving the people they will affect. What is better than a science at the service of the people. 
```

Comment:
```suggestion
It is not easy; there are frictions and conflicting interests among the sectors represented, but what better than to have them at the same table, dialoguing and agreeing on what is best for all?  What is better than knowing that your research and extension efforts serve the people they will affect? What is better than a science at the service of the people? 

This is the way I learned to do science and the way I did science for most of my life.  It's very strange for me to see now science organizations around the world that _aren't_ organized this way and that _don't_ put citizens at the center.

Like the [previous article about cooperatives](/blog/2024-01-05-cooperative/), I write about INTA's governance and share it because it is a feasible model that has been working for 38 years in one of Argentina's largest science and technology institutions.  
```